tests/Mado/QueryBundle/Objects/PagerFantaBuilderTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class PagerFantaBuilderTest extends TestCase
+{
+    private $pagerfantaFactory;
+
+    private $ormAdapter;
+
+    private $pagerfantaBuilder;
+
+    public function testCreateRepresentation()
```

Comment:
May I suggest a verbosa name to make explicit the fact that QB's component is using Hateoas classes? Something like`testCreatePaginatedRepresentationUsingHateoas` or something like that?